packages/mc-scripts/extract-intl.js

```diff
@@ -30,10 +35,11 @@ if (commands.length === 0 || (flags.help && commands.length === 0)) {
   Usage: mc-scripts extract-inl [options] <glob-pattern>..
 
   Options:
-  --output-path         The location where to put the extracted messages
-  --locale=<locale>     (optional) The default locale to use [default "en"]
-  --build-translations  (optional) In case you want to manually build the locale files with the translations [default "false"]
-  --overwrite-core      (optional) By default, if a core.json file exists, existing keys won't be overwritten. This is to ensure that messages in core.json can be updated from external sources. In case you want to avoid this check, you can force writing the extracted messages to core.json [default "false"]
+  --output-path                       The location where to put the extracted messages
+  --locale=<locale>                   (optional) The default locale to use [default "en"]
+  --locales=<locale1,locale2,...>     (optional) The supported locales to map to [default ${supportedLocales.toString()}]
```

Comment:
Nice.

Reply:
This can be used for apps that only require a subset of the locales. By default, we use all our supported locales.